sections/collage.liquid

```diff
@@ -19,10 +19,20 @@
 
 <link rel="stylesheet" href="{{ 'component-deferred-media.css' | asset_url }}" media="print" onload="this.media='all'">
 
+{%- liquid
+  if settings.spacing_grid_horizontal == 0 and settings.spacing_grid_vertical == 0 and settings.card_style == 'card'
+    assign collapsed_class = 'collage--collapsed collage--collapsed-x collage--collapsed-y'
+  elsif settings.spacing_grid_horizontal == 0
+    assign collapsed_class = 'collage--collapsed collage--collapsed-x'
+  elsif settings.spacing_grid_vertical == 0 and settings.card_style == 'card'
+    assign collapsed_class = 'collage--collapsed collage--collapsed-y'
+  endif
+-%}
```

Comment:
Yeah good call out. Must have had tunnel vision with collage. I'm no longer making the exception for card type by using the classes so I think this will be improved but let me know if you still break it.